readthedocs/templates/support/index.html

```diff
@@ -66,13 +66,13 @@ <h2 id="user-support">User Support</h2>
 >
   <div>
     <label for="name">Name</label>
-    <input type="text" name="name" {% if request.user.is_authenticated %} value="{{ request.user.first_name }} {{ request.user.last_name }}" {% endif %}>
+    <input type="text" name="name" value="{{ request.user.get_full_name }}">
     <p></p>
   </div>
 
   <div>
     <label for="email">Email</label>
-    <input type="email" name="email"  {% if request.user.is_authenticated %} value="{{ request.user.email }}" {% endif %}>
```

Comment:
Does this view require login? Is that why this was removed?

Reply:
yeah, this was changed to require login when we hit a lot of spam IRC